### PR TITLE
[5.6] [minor]: SCA, PHP core API usage tweaks

### PR DESCRIPTION
**Pull request by kalessil**

### CODE REVIEW

**Review by carusogabriel — APPROVED** [review]
Interesting plugin :clap:

File: src/Illuminate/Database/Connection.php [hunk 1]
```diff
@@ -504,7 +504,7 @@ public function unprepared($query)
             }
 
             $this->recordsHaveBeenModified(
-                $change = ($this->getPdo()->exec($query) === false ? false : true)
+                $change = ($this->getPdo()->exec($query) !== false)
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: unneeded brackets

File: src/Illuminate/Foundation/Http/Kernel.php [hunk 2]
```diff
@@ -271,7 +271,7 @@ public function hasMiddleware($middleware)
      */
     public function prependMiddleware($middleware)
     {
-        if (array_search($middleware, $this->middleware) === false) {
+        if (! in_array($middleware, $this->middleware)) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Http/Kernel.php]: strict flag?

File: src/Illuminate/Foundation/Http/Kernel.php [hunk 3]
```diff
@@ -286,7 +286,7 @@ public function prependMiddleware($middleware)
      */
     public function pushMiddleware($middleware)
     {
-        if (array_search($middleware, $this->middleware) === false) {
+        if (! in_array($middleware, $this->middleware)) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Http/Kernel.php]: strict flag?

File: src/Illuminate/Routing/Console/ControllerMakeCommand.php [hunk 4]
```diff
@@ -94,7 +94,7 @@ protected function buildClass($name)
         $replace["use {$controllerNamespace}\Controller;\n"] = '';
 
         return str_replace(
-            array_keys($replace), array_values($replace), parent::buildClass($name)
+            array_keys($replace), $replace, parent::buildClass($name)
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/Console/ControllerMakeCommand.php]: are you sure?

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php [hunk 5]
```diff
@@ -1027,8 +1027,8 @@ protected function shouldBlockPhpUpload($value, $parameters)
         }
 
         return ($value instanceof UploadedFile)
-           ? trim(strtolower($value->getClientOriginalExtension())) === 'php'
-           : trim(strtolower($value->getExtension())) === 'php';
+           ? strtolower(trim($value->getClientOriginalExtension())) === 'php'
```
Comment (GrahamCampbell) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: why?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Routing/Console/ControllerMakeCommand.php (hunk 4 above)
Comment (kalessil) [reply src/Illuminate/Routing/Console/ControllerMakeCommand.php]: Yes. Also, tests are green.

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php (hunk 5 above)
Comment (kalessil) [reply src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: strtolower on space characters is not making any sense, hence first make input shorter and then normalize it.

File: src/Illuminate/Routing/Console/ControllerMakeCommand.php (hunk 4 above)
Comment (kalessil) [reply src/Illuminate/Routing/Console/ControllerMakeCommand.php]: But this case should be reverted if maintainers feel more comfortable with the older version. So, if you are in doubt, let's just revert it.

**Comment by kalessil** [comment]
@carusogabriel : thanks =)

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php (hunk 5 above)
Comment (browner12) [reply src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: are there performance implications?  otherwise I see no reason to make this change.

**Pull request closed without merging**